README.md

```diff
@@ -353,6 +353,20 @@ String - Anything that the ssh_authorized_key resource can take for the type att
 
 - *Default*: 'ssh-dss'
 
+purge_ssh_keys
+-----------------
+Boolean - Purge any keys that aren’t managed as ssh_authorized_key resources. This parameter was introduced with Puppet 3.6
+and will break backward compatibility with earlier versions when used. To implicit activate this functionality you need to
+set using_puppet_36 to true additionally.
+
+- *Default*: false
+
+using_puppet_36
```

Comment:
Some minutes later I started to play around with $::puppetversion.
With this code a client prior 3.6.0 will never make use of that functionality:

```puppet
  case versioncmp($::puppetversion, '3.6.0' ) {
    '0','1': {
      $running_puppet_36 = true
    }
    default: {
      $running_puppet_36 = false
    }
  }

  if ($running_puppet_36 == true) and ($using_puppet_36_bool == true) {
    User {
      purge_ssh_keys => $mypurgekey,
    }
  }
```

But it doesn't help while spec testing. They will still fail on Puppet before 3.6.0.
It would need to change the spec tests expectations depending on the executing puppet binary.